### DISCUSSION TITLE
Best way to call Flexget from a python script.

### DISCUSSION THREAD

**Discussion by kingamajick** [Flexget/Flexget#3613] [Q&A]
I have a python script that I would like to call out to Flexget for. I'm wondering whats the best way to do this, is there a method to interact with Flexget directly or is it recommend to call out to Flexget via `subprocess` or similar?

**Comment by lostb1t** [comment]
The best way is to create a custom plugin. Look at the plugins in this repo or any listed here: https://flexget.com/Plugins for inspiration

**Comment by gazpachoking** [comment]
You could also use the rest api, it depends on what exactly you are trying to do.
